packages/@okta/vuepress-site/docs/guides/terraform-oig-resources/main/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Manage Okta Identity Governance resources using Terraform
+meta:
+  - name: description
+    content: Learn how to create, import, and modify Okta Identity Governance resources using Terraform automation.
+layout: Guides
+---
+Learn how to create, import, and modify Okta Identity Governance resources using Terraform automation.
+
+#### Learning outcomes
+
+- Define and create an <StackSnippet snippet="resource-name" inline/> resource using Terraform.
+- Modify an existing <StackSnippet snippet="resource-name" inline/> resource managed by Terraform.
+- Import <StackSnippet snippet="resource-name" inline/> resource from Okta into your Terraform state.
+
+#### What you need
+- Familiarity with the Terraform terms, such as configuration, resources, state, and commands.
+- An Okta org with the OIG SKU enabled.
+- A Terraform configuration that can access your Okta org. See [Okta provider configuration](#okta-provider-configuration).
+- An Okta user account with the super administrator role.
+- Terraform 1.8.5 or later.
+- Okta Terraform provider 6.0.0 or later.
```

Comment:
Based on this changelog (https://github.com/okta/terraform-provider-okta/blob/master/CHANGELOG.md), a lot of OIG resources were added much later. Up until 6.5?